content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,192 @@
 
-Get notified when credits are low
-=================================
+.. example::
+   The following flow focuses on the *SMS* |IAP| service being used from a contact's record.
 
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+   This can be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.
 
-.. image:: in_app_purchase/image4.png
-   :align: center
+   .. image:: in_app_purchase/sms-icon.png
+      :align: center
+      :alt: The SMS icon on a typical contact information form located within an Odoo database.
+
+   One way to utilize the *SMS* |IAP| service with Odoo is showcased in the following steps:
+
+   #. Navigate to the :menuselection:`Contacts application`, and click on a contact with a mobile
+      phone number entered in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the
+      contact form.
+   #. Hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field, and a :guilabel:`📱
+      (phone) SMS` icon appears to the right.
+   #. Click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text Message` pop-up
+      window appears.
+   #. Type a message in the :guilabel:`Message` field of the :guilabel:`Send SMS Text Message`
+      pop-up window. Then, click the :guilabel:`Send SMS` button. Odoo then sends the message, via
+      SMS, to the contact, and logs what was sent in the *chatter* of the contact's form.
+
+   Upon sending the SMS message, the prepaid credits for the *SMS* |IAP| service are automatically
+   deducted from the existing credits. If there are not enough credits to send the message, Odoo
+   prompts the user to purchase more.
+
+.. seealso::
+   For more information on how to use various |IAP| services, and for more in-depth instructions
+   related to SMS functionality in Odoo, review the documentation below:
+
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an |IAP| service is used, the prepaid credits for that service are spent. Odoo prompts
+the purchase of more credits when there are not enough credits left to continue using a service.
+Email alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in *Packs* from the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
```

Comment:
Optional: might be useful to include TIKU's pricing table he unearthed recently for MMC team.